### PR TITLE
update packit config

### PR DESCRIPTION
**Pull request by astoycos**
Start building for stable on PRs so that we
have some coverage.

Propose downsream RPM releases whenever we cut
a release upstream

### CODE REVIEW

**Comment by astoycos** [comment]
So I probably don't want to build-on each PR here since it's pretty slow, but the release bits will still be needed.

File: .packit.yml [hunk 1]
```diff
@@ -35,15 +35,14 @@ jobs:
       - fedora-all
     specfile_path: bpfman.spec
   - job: copr_build
-    trigger: release
-    branch: main
+    trigger: pull_request
+    targets: [fedora-stable]
```
Comment (danielmellado) [inline .packit.yml:39]: nit, I'd format this in a different way. IMHO it enhances readability:
```suggestion
    targets:
      - fedora-stable
```

File: .packit.yml [hunk 2]
```diff
@@ -35,15 +35,14 @@ jobs:
       - fedora-all
     specfile_path: bpfman.spec
   - job: copr_build
-    trigger: release
-    branch: main
+    trigger: pull_request
```
Comment (danielmellado) [inline .packit.yml:38]: if not specified, this would trigger builds for pull request with any target branch.

File: .packit.yml (hunk 1 above)
Comment (danielmellado) [reply .packit.yml:39]: Besides that, I'd be interested in getting to know if we're targeting any specific arch.

File: .packit.yml (hunk 2 above)
Comment (astoycos) [reply .packit.yml:38]: Right I just did that for some testing 
I'll remove that bit.

**Comment by astoycos** [comment]
> Besides the comment, I was thinking that we may want to add a new CI validation for the packit config itself, calling `packit validate-config`

Sounds good what sort of validation are you thinking? Just to make sure no one changes it?

**Review by danielmellado — APPROVED** [review]
lgtm now, thanks!

**Review by Billy99 — APPROVED** [review]
LGTM

**Pull request merged**